demos/index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>First Discovery Tool</title>
+
+        <link rel="stylesheet" href="../src/lib/normalize/normalize.css" />
+        <link rel="stylesheet" href="../src/lib/foundation/css/foundation.min.css" />
+        <link rel="stylesheet" href="../src/css/style.css" />
+    </head>
+
+    <body>
+        <div class="small-12 gpii-tool">
+            <header class="row">
```

Comment:
I'm not sure, semantically, if "header" is appropriate for this content. It's not really header content...